controllers/users.js

```diff
@@ -602,6 +601,31 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const changeUserNickname = async (req, res) => {
+  try {
+    const type = req.body.type;
```

Comment:
Nit: Want to know
what are the data you sending in body object...can you please refer me on API Contract??